building-web-forms/load-from-web-storage/README.md

```diff
@@ -0,0 +1,90 @@
+# Script
+
+In this video, we will save the data collected from user in client side using Web Storage APIs available in HTML5. Since we would like to persist the data even after the browser is closed, we should use `LocalStorage`.
+
+We are collecting `name`, `email`, `password`, `date of birth` and `age group` in our `user form`.
+We will now add a `save` button to persist these data to the LocalStorage.
```

Comment:
We will now add a `save` button to save and persist these data to the LocalStorage.